pyupgrade/_plugins/unittest_aliases.py

```diff
@@ -59,3 +64,15 @@ def visit_Call(
             new=f'self.{method_mapping[node.func.attr]}',
         )
         yield ast_to_offset(node.func), func
+    elif (
+            isinstance(node.func, ast.Attribute) and
+            isinstance(node.func.value, ast.Name) and
+            node.func.value.id == 'unittest' and
+            node.func.attr in FUNCTION_MAPPING
+    ):
+        func = functools.partial(
+            replace_name,
+            name=node.func.attr,
+            new=f'unittest.{FUNCTION_MAPPING[node.func.attr]}',
+        )
+        yield ast_to_offset(node.func), func
```

Comment:
ahah, I'm glad we did this -- now that the code isn't muddled with DRYness it makes the subtle bug way more obvious:

these attributes are only available in a specific version of python -- right now this is performing the rewrite on all target versions

Reply:
> these attributes are only available in a specific version of python -- right now this is performing the rewrite on all target versions

Believe it or not, I thought about that and checked ahead of time! Everything we need seem to exist in 2.7+:

[defaultTestLoader](https://docs.python.org/2.7/library/unittest.html?highlight=defaulttestloader#unittest.defaultTestLoader)

[loadTestsFromModule](https://docs.python.org/2.7/library/unittest.html?highlight=defaulttestloader#unittest.TestLoader.loadTestsFromModule)

[loadTestsFromTestCase](https://docs.python.org/2.7/library/unittest.html?highlight=defaulttestloader#unittest.TestLoader.loadTestsFromTestCase)

[getTestCaseNames](https://docs.python.org/2.7/library/unittest.html?highlight=defaulttestloader#unittest.TestLoader.getTestCaseNames)